Search.cpp

```diff
@@ -188,18 +190,37 @@ void report()
                         {1, 6, 0}};
 
     while (askInput(grid))
-    {
-        State initialState(grid);
+    {   
 
+        State initialState(grid);
+        
+        cout << "Choice a option:\n1 - A* Search\n2 - DFS\n3 - BFS\n4 - Greedy Search\n";
```

Comment:
you can use `cerr` instead of `cout` to display this message on the screen only, if you want to do `a.exe> ​​output.txt`